libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -352,16 +352,25 @@ class YulUtilFunctions
 
 	/// @returns the name of a function that allocates memory.
 	/// Modifies the "free memory pointer"
-	/// Arguments: size
-	/// Return value: pointer
+	/// signature: (size) -> ptr
 	std::string allocationFunction();
 
-	/// @returns the name of the function that allocates temporary memory with predefined size
+	/// @returns the name of the function that allocates memory whose size will be defined later,
+	/// or it will be released. The allocation can be finalized using finalizeAllocationFunction.
+	/// Any other allocation will invalidate the memory pointer unless finalizeAllocationFunction
+	/// is called.
 	/// Return value: pointer
```

Comment:
Return value was removed in other comments so I think it should be removed here too.
```suggestion
```